cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -125,9 +125,7 @@ pAddressCmd =
       , Opt.command "build"
           (Opt.info pAddressBuild $ Opt.progDesc "Build a Shelley payment address, with optional delegation to a stake address.")
       , Opt.command "build-multisig"
-          (Opt.info pAddressBuildMultiSig $ Opt.progDesc "Build a Shelley payment multi-sig address.")
-      , Opt.command "build-script"
-          (Opt.info pAddressBuildScript $ Opt.progDesc "Build a Shelley script address.")
+          (Opt.info pAddressBuildScript $ Opt.progDesc "Build a Shelley payment multi-sig address.")
```

Comment:
I think this should still be called `build-script` for building a script address. It just so happens that, right now, we only have multisignature scripts.